Dockerfile

```diff
@@ -48,15 +39,13 @@ ENV LC_ALL en_ZA.UTF-8
 ENV LC_CTYPE en_ZA.UTF-8
 ENV LC_MESSAGES en_ZA.UTF-8
 
+# Installing kubernetes-specific python packages
+RUN pip install apache-airflow[kubernetes]==${AIRFLOW_VERSION} \
+    && pip install flask_bcrypt \
+    && pip install Flask-OAuthlib
+
 # Changing back to the airflow user
 USER airflow
 
-# Creating a DAGs dir in the airflow user's directory
-RUN mkdir /usr/local/airflow/dags
-
-# Using a local executor, and unloading the examples
-ENV EXECUTOR "Local"
```

Comment:
Is the executor Local in the k8s version? If so, might be nice to include the `ENV EXECUTOR "Local" along with a comment as an easter egg for others, given how hard this was for us.

Reply:
A fair point, this lives in the `airflow-values.yaml` file now, too.

I'll add the comment, but I'm going to leave it here, on the outside chance someone is going to run the Airflow standalone to the k8s cluster.